src/test/kotlin/StringAddCalculatorTest.kt

```diff
@@ -0,0 +1,60 @@
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatExceptionOfType
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.NullAndEmptySource
+import org.junit.jupiter.params.provider.ValueSource
+
+class StringAddCalculatorTest {
+
+    private lateinit var calculator: StringAddCalculator
+
+    @BeforeEach
+    fun setUp() {
+        calculator = StringAddCalculator
+    }
+
+    @DisplayName(value = "빈 문자열 또는 null 값을 입력할 경우 0을 반환해야 한다.")
+    @ParameterizedTest
+    @NullAndEmptySource
+    fun emptyOrNull(text: String?) {
+        assertThat(calculator.add(text)).isZero
+    }
```

Comment:
```suggestion
    @ParameterizedTest
    @NullAndEmptySource
    fun `빈 문자열 또는 null 값을 입력할 경우 0을 반환해야 한다.`(text: String?) {
        assertThat(calculator.add(text)).isZero
    }
```
기본적으로는 테스트명을 함수 이름으로 하는건 어떨까요?